### DISCUSSION TITLE
Service Pooler

### DISCUSSION THREAD

**Discussion by lucianoalien** [OpenSourceCommunityBrasil/REST-DataWare#90] [Q&A] [answered]
Olá, eu fiz a atualização passo a passo, mas não estou conseguindo encontrar alguns componentes como ServicePooler

**Comment by mobius1qwe** [comment, marked as answer]
Atualizou da 1.4.3 pra 2.0? segue aqui: https://github.com/OpenSourceCommunityBrasil/REST-DataWare/wiki/Atualiza%C3%A7%C3%A3o-da-vers%C3%A3o-1.4.x-para-a-vers%C3%A3o-2.0-Delphi

Atualizou da 2.0 pra alguma outra 2.0? segue aqui: https://github.com/OpenSourceCommunityBrasil/REST-DataWare/wiki/Atualizar-da-vers%C3%A3o-2.0--pra-uma-mais-nova

É Lazarus? segue aqui: https://github.com/OpenSourceCommunityBrasil/REST-DataWare/wiki/LazarusUpd
